crategen/models.py

```diff
@@ -0,0 +1,96 @@
+from pydantic import BaseModel, Field, validator,root_validator
+from typing import List, Optional, Dict, Any
+
+class Executor(BaseModel):
+    image: str
+    command: List[str]
+
+class TESInputs(BaseModel):
+    url: str
+    path: str
+
+class TESOutputs(BaseModel):
+    url: str
+    path: str
+
+class TESLogs(BaseModel):
+    end_time: Optional[str] = None
+
+class TESData(BaseModel):
+    id: str
+    name: str
+    description: Optional[str] = ""
+    executors: List[Executor]
+    inputs: List[TESInputs]
+    outputs: List[TESOutputs]
+    creation_time: str
+    logs: List[TESLogs]
+
+class WESRunLog(BaseModel):
+    name: Optional[str] = None
+    start_time: Optional[str] = None
+    end_time: Optional[str] = None
+
+class WESOutputs(BaseModel):
+    location: str
+    name: str
+
+class WESData(BaseModel):
+    run_id: str
+    run_log: WESRunLog
+    state: str
+    outputs: List[WESOutputs]
+
+    @root_validator(pre=True)
```

Comment:
Thank you for the feedback. I've added extra = "forbid" to the models for better readability and error handling. I'll ensure all required fields for WES Data are included to prevent errors when running the CLI with wes_full.json.